Gemfile

```diff
@@ -6,10 +6,12 @@ gem "haml", "~> 5.2"
 
 gem "cucumber", "~> 3.2"
 gem "citizens-advice-style", github: "citizensadvice/citizens-advice-style-ruby", tag: "v0.3.0"
+gem "rubocop", "~> 0.85.1"
```

Comment:
This will conflict with other things as / when we upgrade. So it's best to rely on ca-style gem.

It also can be updated to v0.4 (Think I just did that though?)